app/src/main/res/values/strings.xml

```diff
@@ -146,6 +146,7 @@
 
         This app is just one piece of a bigger series of apps. You can find the rest of them at http://www.simplemobiletools.com
     </string>
+    <string name="one_finger_zoom">Toggle one finger zoom</string>
```

Comment:
move this at the settings section, right under manage_extended_details. Add it to every language file, even if in english obviously. Ye, I have to keep adding it like that too :/